docs/config/dtable_db_conf.md

```diff
@@ -35,8 +35,7 @@ This sections defines where the database files for bases with activated big data
 This sections contains options for caching bases from dtable-server.
 
 - `dtable_server_url`: local address for dtable-server. You must specify this option. Typically it is `http://127.0.0.1:5000`.
-- `total_cache_size`: How much memory shall be used for caching bases. After this threshold is reached, cached bases will be cleaned with LRU algorithm. Cleaning stops when memory consumption reduces to 70% of this threshold. Unit is in MB. Defaults to 500 (=500 MB).
-- `clean_cache_interval`: Interval between cache cleaning. Unit is in seconds. Defaults to 300.
+- `total_cache_size`: size of the cache in MB. Defaults to 500. Prior to Enterprise Edition 4.4, the size of base objects in cache was measured by storage size, and the cache was periodically cleaned to 70% of total_cache_size. Since Enterprise Edition 4.4, the size is measured by memory consumption (about 8 times bigger than before), and the cache will be cleaned with LRU algorithm to fit total_cache_size.
```

Comment:
这个默认值应该改为 2GB 吧，相应的代码也改一下。

后面的不用写太多技术细节，可以改为这样：size of the base cache in MB. Since 4.4 version, the default is 2000MB (2GB). Before 4.4 version, the default is 500MB. This difference is due to a change of memory consumption estimation algorithm. With 4.4 version the new algorithm is more close to the real memory consumption of the cache.